settings.py

```diff
@@ -16,7 +16,8 @@
 # Background Image Settings. Lower Alpha is more transparent
 BG_IMG = "assets/fib_bg1.png"
 IMG_ALPHA = 128
-BABY_MODE = True
+# BABY_MODE = True
```

Comment:
We should remove unused comments.
```suggestion
```

Reply:
Good Suggestion. But you're still a baby. I thought we turned that mode off XD